OracleLinuxDevelopers/oraclelinux7/python/3.6-oracledb/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 FROM oraclelinux:7-slim
 
-RUN yum -y install oracle-release-el7 oraclelinux-developer-release-el7 && \
+RUN yum -y install oracle-release-el7 oraclelinux-developer-release-el7 oracle-instantclient-release-el7 && \
```

Comment:
I guess this should have removed oracle-release-el7

Reply:
I agree.

Reply:
The tidy-up is in https://github.com/oracle/docker-images/pull/1985